.github/dependabot.yml

```diff
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
```

Comment:
@malept do we want to check on a daily basis?

Reply:
I don't think it matters too much.